docs/new/posts/Percona Operators/operator-for-postgresql-2.6.0.md

```diff
@@ -0,0 +1,42 @@
+---
+date: 2025-03-12
+description: >
+  Percona Operator for PostgreSQL 2.6.0 has been released on March 12, 2025.
+authors: [nastena1606]
+categories:
+  - Percona Operators
+tags:
+  - Percona Operator for PostgreSQL
+---
+
+# Percona Operator for PostgreSQL 2.6.0 has been released
+
+<!-- more -->
+
+[Percona Operator for PostgreSQL](https://docs.percona.com/percona-operator-for-postgresql/2.0/){:target="_blank"} 2.6.0 has been released on March 12, 2025.
+
+Try it out using the [Quickstart guide](https://docs.percona.com/percona-operator-for-postgresql/2.0/quickstart.html){:target="_blank"}.
+
+New features and improvements introduced by this release include:
+
+* Backup improvements:
+
+    * An improved backup logic now allows retrying a failed backup in the same backup Pod for a specified number of times before deleting this Pod and creating a new one. This is beneficial in case of short connectivity issues or timeouts.   
```

Comment:
```suggestion
    * An improved backup logic now allows [retrying a failed backup](https://docs.percona.com/percona-operator-for-postgresql/2.0/operator.html#backupspgbackrestjobsbackofflimit){:target="_blank"} in the same backup Pod for a specified number of times before deleting this Pod and creating a new one. This is beneficial in case of short connectivity issues or timeouts.   
```